models/balancer/arbitrum/balancer_v2_arbitrum_transfers_bpt.sql

```diff
@@ -0,0 +1,48 @@
+{{
+    config(
+        alias='transfers_bpt',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date', 'evt_tx_hash', 'evt_index'],
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "project",
+                                    "balancer_v2",
```

Comment:
there's a bit of mismatch of where `balancer` or `balancer_v2` will be used. how important is that?
examples:
schema: `balancer_arbitrum` based on dbt project file (of course, blockchain will change across each)
dune UI explorer: `balancer_v2` under spells at root level, where each spell will live within, using schema name above within
file names: `balancer_v2`

i'd suggest matching schema and explorer UI display, meaning you'd change either the value on this line (for all blockchains) or the schema names in dbt_project yaml file.

Reply:
Not sure if I understood, but these views are related to Balancer V2, so I would like to select them by `balancer_v2_blockchain_transfers_bpt` and they should appear under `balancer_v2` spells.

Reply:
got it -- let me push a change and explain how that is covered 👍

Reply:
nvm, i don't have access to push. i'll try to comment where you can update.

Reply:
due to [this block](https://github.com/duneanalytics/spellbook/blob/main/dbt_project.yml#L42) of yaml code in the dbt_project file, all models built within these directories (i.e. `models/balancer/arbitrum/`) will have a default schema name of `balancer_arbitrum`. 

in spellbook, contract versions are the most granular level of models, so if we want to override the default value (i.e. `balancer_arbitrum`) to a versioned schema, we'd apply a `schema` property in the individual model(s) config block at the top of each. [here](https://github.com/duneanalytics/spellbook/pull/2056/files#diff-8d26d52e8cfb68542e7a2cdba56701b15a17d8091088ab8f6ce631469e66cb2cR3) is where you'd do for arbitrum. above alias (i.e. table name), just add a `schema` property and assign the string value `balancer_v2_arbitrum`. each blockchain would follow a similar approach.

since we handle versions within models, there is no need to edit the dbt_project file. that will stay the the directory level.

Reply:
those should be the only changes for each blockchain model. the `expose_spell` macro [here](https://github.com/duneanalytics/spellbook/pull/2056/files#diff-8d26d52e8cfb68542e7a2cdba56701b15a17d8091088ab8f6ce631469e66cb2cR11) on each model already uses `balancer_v2` -- which means the dune UI explorer will go:
`spells --> balancer_v2 --> transfers_bpt` with the blockchain logo next to each.

Reply:
I tried to add one line in the config block in each .sql file, follow the above instruction.

Reply:
it's clear to me now, thanks for the thorough explanation @jeff-dude! 🙏